addons/website/static/src/js/menu/edit.js

```diff
@@ -118,29 +177,89 @@ var EditPageMenu = websiteNavbarData.WebsiteNavbarActionWidget.extend({
         return res;
     },
     /**
+     * Redirects the user to the same page in translation mode (or start the
+     * translator is translation mode is already enabled).
+     *
      * @private
+     * @returns {Promise}
      */
-    _createWysiwyg: async function () {
+    _startTranslateMode: async function () {
+        // Add class in navbar and hide the navbar.
+        this.trigger_up('edit_mode');
+
+        if (!this._mustEditTranslations) {
+            window.location.search += '&edit_translations';
+            return new Promise(function () {});
+        }
+
+        if (!localStorage.getItem(localStorageNoDialogKey)) {
+            new TranslatorInfoDialog(this).open();
+        }
+
+        this.wysiwyg = await this._createWysiwyg({
+            snippets: null,
+            toolbar: false,
+            customCommands: {
+                saveOdoo: {handler: this._onSaveTranslation.bind(this)}
+            },
+            mode: {
+                id: 'translate',
+                rules: [
+                    {
+                        selector: [],
+                        editable: false,
+                    },
+                    {
+                        selector: [this.JWEditorLib.ContainerNode],
+                        breakable: false,
+                    },
+                    {
+                        selector: [node => !!node.modifiers.find(this.JWEditorLib.OdooTranslationFormat)],
+                        editable: true,
+                    },
+                    {
+                        selector: [node => {
+                            const attributes = node.modifiers.find(this.JWEditorLib.Attributes);
+                            return attributes && attributes.classList.has('o_not_editable');
+                        }, () => true],
+                        editable: false,
+                    },
+                ],
```

Comment:
Same as above, I would hide this in `wysiwyg.js`

Reply:
Same answer as above.